## use in both C++ and Python
uint8 MoveToPose = 0
uint8 ConstantVelocity = 1
uint8 AdmittanceMotion = 2
uint8 Displacement = 3
